script.js

```diff
@@ -1,5 +1,202 @@
+When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6.
+The player picks the order of the dice they want. For example, if they wanted the number 63, they would specify that the 2nd dice goes first. You can choose how the player specifies dice order.
+After both players have rolled and chosen dice order, the player with the higher combined number wins.
+*/
+
+// Global Var
+player1diceResults = []
+player2diceResults = []
+player1 = ""
+player2 = ""
+player1Number = 1
+player2Number = 2
+// 
+
+
+var rollDice = function(){
+  // Create a dice rolling from 1 - 6
+  var randomDice = Math.floor(Math.random() * 6) + 1
+  console.log(`Dice roll: ${randomDice}`)
+  return randomDice
+}
+
+var roll2Dice = function(array){
+  firstDice = rollDice()
+  secondDice = rollDice()
+  array.push(firstDice)
+  array.push(secondDice)
+  console.log(`roll2Dice: ${firstDice}, ${secondDice}`)
+  console.log(`array: ${array}`)
+  console.log(`player1diceResults: ${player1diceResults}`)
+  console.log(`player2diceResults: ${player2diceResults}`)
+  return array
+}
+
+var playerTurn = function(playerNumber,choice,playerArray){
+  if (playerNumber == 1 && choice == null){
+    player1diceResults = roll2Dice(player1diceResults)
+    myOutputValue = `Player ${playerNumber}, Here is your rolled dice result: <br><br>
+    dice 1 : ${player1diceResults[0]} <br><br>
+    dice 2 : ${player1diceResults[1]} <br><br>
+    Please choose the order of desired dice result. <br>
+    Type 1 and click enter if you wish for dice 1 to be the first number <br>
+    Type 2 and click enter if you wish for dice 2 to be the first number`;
+    return myOutputValue
+  }
+  else if (playerNumber == 2 && choice == null){
+    player2diceResults = roll2Dice(player2diceResults)
+    myOutputValue = `Player ${playerNumber}, Here is your rolled dice result: <br><br>
+    dice 1 : ${player2diceResults[0]} <br><br>
+    dice 2 : ${player2diceResults[1]} <br><br>
+    Please choose the order of desired dice result. <br>
+    Type 1 and click enter if you wish for dice 1 to be the first number <br>
+    Type 2 and click enter if you wish for dice 2 to be the first number`;
+    return myOutputValue
+  }
+  else if (choice == 1){
+    myOutputValue = `${playerNumber} have chose dice 1 to be the first in order, your result is ${playerArray[0]}${playerArray[1]} <br><br>
+    Click enter for next.`;
+
+    console.log(`player ${playerNumber} dice results: ${playerArray[0]}${playerArray[1]}`);
+    console.log(`user dice order: ${choice}`);
+    return myOutputValue
+  }
+  else if (choice == 2){
+    myOutputValue = `${playerNumber} have chose dice 2 to be the first in order, your result is ${playerArray[1]}${playerArray[0]} <br><br>
+    Click enter for next.`;
+
+    console.log(`player ${playerNumber} dice results: ${playerArray[1]}${playerArray[0]}`);
+    console.log(`user dice order: ${choice}`);
+    return myOutputValue
+  }
```

Comment:
this part prob needs additional input validation. when trying to play your game, if i enter a choice that is not 1 or 2, i get back `undefined` in the output box which is probably not what you want